frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx

```diff
@@ -93,6 +117,11 @@ async function setup({
 }: SetupOpts = {}) {
   if (getActionIsEnabledInDatabase(dashcard)) {
     fetchMock.post(ACTION_EXEC_MOCK_PATH, { "rows-updated": [1] });
+
+    // for ActionCreator modal (action edit modal)
+    setupDatabasesEndpoints([database]);
+    setupCardsEndpoints([card, { ...card, id: ACTION_MODEL_ID }]);
+    fetchMock.get(`path:/api/action/${ACTION.id}`, ACTION);
```

Comment:
There should be an existing helper for action endpoints. If there is none, let's create it in `server-mocks`